timescaledb/how-to-guides/page-index/page-index.js

```diff
@@ -7,14 +7,44 @@ module.exports = [
       {
         title: "Install TimescaleDB",
         href: "install-timescaledb",
+        metaTags: [
```

Comment:
The result would be this: 

![Uploading Screen Shot 2021-06-09 at 08.44.27.png…]()

Reply:
??